### PR TITLE
Initialize app

### PR DESCRIPTION
**Pull request by Mucyosoda**
**Bookstore**
This project will lay the foundation's Bookstore website and was created in React, Redux, babelrc, linter and webpack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## bookstore
+## Book store
```
Comment (dasileker) [inline README.md:5]: - [ ] Please remove all generated readme file details and additional details about the project.

File: src/Redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (dasileker) [inline src/Redux/books/books.js:1]: - [ ] the **redux/book** should be empty and this component should be added in the components folder + prop.


kkk

File: src/Redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CategoriesPage = () => (
+  <>
+    <p className="text">Under Construction</p>
+  </>
+);
+
```
Comment (dasileker) [inline src/Redux/categories/categories.js]: - [ ] The same thing goes for the category.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Mucyosoda,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Redux/books/books.js (hunk 2 above)
Comment (Mucyosoda) [reply src/Redux/books/books.js:1]: Hello dasileker,
Thank you for your feedback, and I understand changes required but according to the project requirement some requirements will be applied in the next project, here was just initialization

**Review by omar-labana — APPROVED** [review]
Hi @Mucyosoda, 

- [ ] _[OPTIONAL]_ please update your README file before moving to the next milestone.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**